GenerateGraph.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import json
+from typing import AnyStr, List
+
+from DbInterface import Neo4jStream
+from TestGraphGenerator import export_database_neo4j, create_graph_map, export_database_json
+from TestGraphGenerator.DbTranformations import import_database_json
+from TestGraphGenerator.DbTranformations.DbLoader import delete_database_neo4j
+
+
+def load_names_data_set(path: AnyStr) -> List[AnyStr]:
+    """
+    Loads a list of names from a json file to use as properties in the test database.
+    :param path: the path to the file containing the names
+    :return: A list of names if the file is readable else returns none
+    """
+    with open(path) as data_file:
+        data_json = json.load(data_file)
```

Comment:
No validation. 
Could throw a json parse error
If you want the error, at least add logs or something